tools/topology/topology2/production/tplg-targets-ace3.cmake

```diff
@@ -44,6 +44,15 @@ DEEPBUFFER_FW_DMA_MS=10,DEEP_BUF_SPK=true"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+"cavs-sdw\;sof-ptl-rt712-l3-rt1320-l3\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=1,NUM_DMICS=4,\
```

Comment:
If it has PCH DMIC, the tplg name should be with `-4ch` suffix. Or maybe we should create 2 topologies 1 is with PCH DMIC and the other isn't.